build.sh

```diff
@@ -49,12 +49,15 @@ args=($temp)
 
 dockerbuild()
 {
-    BUILD_COMMAND=/opt/code/build_projects/dotnet-cli-build/build.sh $DIR/scripts/dockerrun.sh --non-interactive "$@"
+    # Initialize the tooling and build
+    BUILD_COMMAND="/opt/code/init-tools.sh && /opt/code/build_projects/dotnet-cli-build/build.sh" $DIR/scripts/dockerrun.sh --non-interactive "$@"
```

Comment:
You shoulnd't need this nor the change below either. init-tools is being invoked by build.sh already.